ktor-server/ktor-server-plugins/ktor-server-caching-headers/jvm/src/io/ktor/server/plugins/cachingheaders/CachingHeaders.kt

```diff
@@ -12,80 +12,62 @@ import io.ktor.util.*
 import io.ktor.util.pipeline.*
 
 /**
- * A plugin that adds the capability to configure the Cache-Control and Expires headers using [CachingOptions].
- * It invokes [optionsProviders] for every response and use first non-null [CachingOptions]
+ * A configuration for the [CachingHeaders] plugin
  */
-public class CachingHeaders private constructor(
-    private val optionsProviders: List<(OutgoingContent) -> CachingOptions?>
-) {
+public class CachingHeadersConfiguration {
```

Comment:
CachingHeadersConfiguration -> CachingHeadersConfig

Reply:
Thanks for your help, it contains plugin rewriting, but without test rewriting